how-to-guides/data-versioning/notebooks/Compare_model_training_runs_on_dataset_versions.ipynb

```diff
@@ -91,7 +88,7 @@
     ]
    },
    "source": [
-    "As an example I'll use a script that trains an Scikit-learn model on iris dataset."
+    "As an example, I will use a script that trains a scikit-learn model on the iris dataset."
```

Comment:
First person singular should be avoided in docs and examples. "We" is acceptable in some cases